internal/types/definitions.go

```diff
@@ -1,15 +1,23 @@
 package types
 
 import (
+	"net/http"
+
 	client "github.com/pingidentity/pingfederate-go-client/v1200/configurationapi"
 	"github.com/pingidentity/terraform-provider-pingfederate/internal/version"
 )
 
 // Configuration used by the provider and resources
 type ProviderConfiguration struct {
```

Comment:
Maybe we could have this struct just hold the contexts themselves, rather than the vars to create the contexts? Not 100% on whether there would be any issues with that.

Reply:
I'll add a GA case for this.